packages/suite/src/components/suite/NotificationCard.tsx

```diff
@@ -22,34 +15,46 @@ interface NotificationCardProps {
     ['data-test']?: string;
 }
 
-const getMainColor = (variant: NotificationCardProps['variant'], theme: SuiteThemeColors) => {
+const getTextColor = (variant: NotificationCardProps['variant'], theme: DefaultTheme) => {
     switch (variant) {
```

Comment:
How about using the `ButtonVariant` as the `variant` type?

Reply:
```
export type ButtonVariant =
    | 'primary'
    | 'secondary'
    | 'tertiary'
    | 'info'
    | 'warning'
    | 'destructive';
```

We have only: 
```
    | 'info'
    | 'warning'
    | 'destructive'
```

Reply:
https://github.com/trezor/trezor-suite/issues/10818